### PR TITLE
ref(js): Convert Gravatar to a FC

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/avatar/gravatar.tsx [hunk 1]
```diff
@@ -15,75 +15,62 @@ type Props = {
...
+  onLoad,
+  suggested,
+  grayscale,
+}: Props) {
+  const isMounted = useRef(false);
+  const {gravatarBaseUrl} = useLegacyStore(ConfigStore);
+  const [MD5, setMD5] = useState<HasherHelper>();
+
+  const loadMd5Helper = useCallback(async () => {
+    const mod = await import('crypto-js/md5');
```
Comment (malwilley) [inline static/app/components/avatar/gravatar.tsx:35]: Seems like this would be a bit simpler if it used Suspense instead?

**Review by malwilley — COMMENTED** [review]
Changes look fine but some tests are failing

File: static/app/components/avatar/gravatar.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/components/avatar/gravatar.tsx:35]: Oh yea

Comment (evanpurkhiser) [reply static/app/components/avatar/gravatar.tsx:35]: this is def pre suspense, good call

**Comment by evanpurkhiser** [comment]
@malwilley let's keep the awaited import like this for now actually

**Review by malwilley — APPROVED** [review]

**Pull request merged**